tutorial/gallery.py

```diff
@@ -203,8 +203,8 @@ def AppSection(app_name,
 
         AppSection(
             app_name='NYC Uber Rides',
-            app_link='https://dash-gallery.plotly.host/dash-uber-rides/',
-            code_link='https://github.com/plotly/dash-uber-rides-demo',
+            app_link='https://dash-gallery.plotly.host/dash-uber-rides-demo/',
+            code_link='https://github.com/plotly/dash-sample-apps/tree/master/apps/dash-uber-rides-demo',
             img_src='assets/images/gallery/uber-rides.gif',
```

Comment:
```suggestion
            img_src='assets/images/gallery/uber-rides.gif',
```
Sorry, forgot to mention this. Please update the gif too!